sdks/python/apache_beam/io/filebasedsource.py

```diff
@@ -49,6 +48,13 @@
 
 __all__ = ['FileBasedSource']
 
+try:
+  unicode       # pylint: disable=unicode-builtin
```

Comment:
How about 

```python
from past.builtins import long, unicode
```

This applies to other similar try-except imports in this (and related) PRs.

Reply:
I used the try/except method to be consistent with the methodology implemented in #5053 and outlined in [the doc](https://docs.google.com/document/d/1xDG0MWVlDKDPu_IW9gtMvxi2S9I0GB0VDTkPhjXT0nE/edit#heading=h.ir03vqq4rotn) by @RobbeSneyders. I believe there where some issues in the coder package for the typechecks using `past.utils`

Reply:
I'd humbly suggest reconsidering (unless there are indeed issues with using `past` in some modules) as try-except is much more verbose syntactically.

Reply:
The try/except was indeed used to prevent problems with typechecking when using `builtins` from the future package.
However, I tested this again, and it seems that there are no such problems using the `past.builtins`.

Reply:
I have created the issue [BEAM-4730](https://issues.apache.org/jira/browse/BEAM-4730) to apply this change to already ported packages.

Reply:
This is done in #5869.

Reply:
Great, thank you!